tests/gtest/avifrgbtoyuvtest.cc

```diff
@@ -307,7 +308,9 @@ TEST_P(RGBToYUVTest, ConvertWholeBuffer) {
   // max_abs_average_diff is not tested here because it is not meaningful for
   // only 3*3 conversions as it takes the maximum difference per conversion.
   // PSNR is averaged on all pixels so it can be tested here.
-  EXPECT_GE(GetPsnr(sq_diff_sum, num_diffs, rgb_max), min_psnr);
+  EXPECT_GE(GetPsnr(static_cast<double>(sq_diff_sum),
+                    static_cast<double>(num_diffs), rgb_max),
+            min_psnr);
```

Comment:
libavif\tests\gtest\avifrgbtoyuvtest.cc(310): warning C4244: 'argument': conversion from 'int64_t' to 'double', possible loss of data